<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalEdit</class>
 <widget class="QWidget" name="JournalEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Journal</string>
  </property>
  <widget class="QTabWidget" name="tabWidget18">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>581</width>
     <height>310</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Common</string>
    </attribute>
    <widget class="QLabel" name="textLabel4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>76</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="eDescription">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>127</y>
       <width>551</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="eName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>500</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>43</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cbType">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Common</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Special</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Othes</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="TabPage">
    <attribute name="title">
     <string>Documents</string>
    </attribute>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <widget class="QPushButton" name="bAddDoc">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>26</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/arrow_left.png</normaloff>:/images/arrow_left.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="bRemoveDoc">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>70</y>
          <width>26</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/editdelete.png</normaloff>:/images/editdelete.png</iconset>
        </property>
       </widget>
       <widget class="QTableWidget" name="journalDocs">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>261</height>
         </rect>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <column>
         <property name="text">
          <string>Documents</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="allDocs">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>0</y>
          <width>251</width>
          <height>261</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <column>
         <property name="text">
          <string>Documents</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Columns</string>
    </attribute>
    <widget class="QTreeView" name="columnsDoc">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>31</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="bAddCol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/arrow_left.png</normaloff>:/images/arrow_left.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="bRemoveDocCol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/editdelete.png</normaloff>:/images/editdelete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="bMoveUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/arrow_up.png</normaloff>:/images/arrow_up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="bMoveDown">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/images/arrow_down.png</normaloff>:/images/arrow_down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="bNewCol">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ananasprojectmanager/libananas/designer.qrc">
          <normaloff>:/add.png</normaloff>:/add.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTreeView" name="treeDocs">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>256</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Aliases</string>
    </attribute>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tAliases">
       <column>
        <property name="text">
         <string>Lang</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="TabPage">
    <attribute name="title">
     <string>Access</string>
    </attribute>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tRoles">
       <column>
        <property name="text">
         <string>Read</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Write</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="Action">
   <property name="iconText">
    <string>Action</string>
   </property>
   <property name="name" stdset="0">
    <cstring>Action</cstring>
   </property>
  </action>
  <action name="Action_2">
   <property name="iconText">
    <string>Action_2</string>
   </property>
   <property name="name" stdset="0">
    <cstring>Action_2</cstring>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="../ananasprojectmanager/libananas/designer.qrc"/>
 </resources>
 <connections/>
</ui>
